### DISCUSSION TITLE
Why Rust

### DISCUSSION THREAD

**Discussion by rdabc** [nkaz001/hftbacktest#115] [General]
Hi,

Why choose Rust over Cython, C++, or other languages in this project?
Thank you,

**Comment by nkaz001** [comment]
Implementing backtesting in C++ is not much different from Rust, but implementing a connector to a crypto exchange based on a web API is more challenging. In Rust, I can utilize excellent libraries, especially for async web APIs, to implement a live bot.  Additionally, Rust aligns more with my personal interests.
